server/src/model/user.py

```diff
@@ -6,10 +6,12 @@
 from typing import Optional, Tuple, cast
 
 from flask_login import UserMixin
-from mongoengine import Document, StringField
+from mongoengine import Document, ObjectIdField, StringField
+from src.clients.google import get_user_info
+from src.model.family import Family
+from src.utils.error import AppError, ErrorCode, error_bounded
 
-from ..clients.google import get_user_info
-from ..utils.error import AppError, ErrorCode, error_bounded
+UNASSIGNED_FAMILY_INTERVAL = 365  # Unassigned families auto-assigned to 1-yr interval
```

Comment:
imo, we should make this 10yrs so ppl dont actually use it. A yearly family may be realistic.